docs/pages/play.vue

```diff
@@ -257,6 +279,73 @@ export default {
       // Check if editors contain default JS and Template
       return this.js.trim() === defaultJS.trim() && this.html.trim() === defaultHTML.trim()
     },
+    codepen_data() {
+      const data = {
+        editors: '101',
+        layout: 'left', // left, right, top
+        html_pre_processor: 'none',
+        css_pre_processor: 'none',
+        css_prefix: 'autoprefixer',
+        js_pre_processor: 'babel',
+        head: '<meta name="viewport" content="width=device-width">',
+        css_external: [
+          '//unpkg.com/bootstrap/dist/css/bootstrap.min.css',
+          '//unpkg.com/bootstrap-vue@latest/dist/bootstrap-vue.css'
+        ].join(';'),
+        js_external: [
+          '//unpkg.com/babel-polyfill@latest/dist/polyfill.min.js',
+          '//unpkg.com/vue@latest/dist/vue.min.js',
+          '//unpkg.com/bootstrap-vue@latest/dist/bootstrap-vue.js'
+        ].join(';'),
+        html: this.fiddle_html,
+        js: this.fiddle_js,
+        css: 'body { padding: 1rem; }'
+      }
+      return JSON.stringify(data)
+    },
+    codesandbox_data() {
+      const html = this.html.trim() || '<div></div>'
+      const js = this.js.trim() || '{}'
+      const vueContent = dedent`
+        <template>
+          ${html}
+        </template>
+
+        <script>
+          export default ${js}
+        <\/script>
+      `.replace('\\/', '/')
+      const htmlContent = '<div id="app"></div>'
+      const jsContent = dedent`
```

Comment:
Anyway we can pass it the basic CSS taht gives body a bit of padding (like in jsFiddle and CodePen options)?

The sandbod results have all the content touching both sides of the result window:
![image](https://user-images.githubusercontent.com/2781561/55865104-1e956b00-5b54-11e9-913a-996f718ddf7f.png)

Reply:
Done